geopandas/io/tests/test_sql.py

```diff
@@ -363,6 +363,128 @@ def test_read_postgis_chunksize(self, connection_postgis, df_nybb):
         # by user; should not be set to 0, as from get_srid failure
         assert df.crs is None
 
+    @pytest.mark.parametrize("connection_postgis", POSTGIS_DRIVERS, indirect=True)
+    def test_read_postgis_multiple_geometry_columns(self, connection_postgis, df_nybb):
```

Comment:
I wasn't sure if it would be preferred to split this up into multiple tests and add the creation of a postgis table with multiple geometry columns in geopandas.tests.util.py in the way create_postgis is formulated there.

Ended up doing it all here since we aren't really testing the writing of the table and only the reading and processing of geometry columns. Let me know if you'd like me to go down the other route with multiple smaller tests re-creating the table each time.